slither/slither.py

```diff
@@ -93,16 +94,25 @@ def __init__(self, target: Union[str, CryticCompile], **kwargs):
         except InvalidCompilation as e:
             # pylint: disable=raise-missing-from
             raise SlitherError(f"Invalid compilation: \n{str(e)}")
+
         for compilation_unit in crytic_compile.compilation_units.values():
             compilation_unit_slither = SlitherCompilationUnit(self, compilation_unit)
             self._compilation_units.append(compilation_unit_slither)
-            parser = SlitherCompilationUnitSolc(compilation_unit_slither)
+            parser = SlitherCompilationUnitSolc(compilation_unit_slither, False)
```

Comment:
For readability, I don't recommend passing a boolean value (unless it's a real boolean operator like `and`), since reader can't know what that means.
In Python, we can use keyword-argument to make it more readable like `generates_certik_ir=False` here.